user/TASK/gimbal_task/gimbal_task.c

```diff
@@ -168,20 +225,7 @@ int get_vision_signal(void) {
  */
 void send_to_uart(Gimbal_t *gimbal_msg) 	
```

Comment:
Put the print/debug statements back

Reply:
I think not in master...

Reply:
I thought send_to_uart() was supposed to act like a debug print for that task, printing out all info it can. Normally, it would be commented out in the task loop so as not to clog uart.